src/tests/CMakeLists.txt

```diff
@@ -93,7 +93,17 @@ if(ENABLE_TESTING)
     target_compile_definitions(unit_${PROG} PRIVATE BOOST_TEST_DYN_LINK)
     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Run_unit_${PROG})
     add_test(NAME unit_${PROG} COMMAND $<TARGET_FILE:unit_${PROG}> WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Run_unit_${PROG})
-    # run tests for xtp (for coverage) as well
     set_tests_properties(unit_${PROG} PROPERTIES LABELS "xtp;votca;unit")
   endforeach(PROG)
-endif()
+
+  add_test(
+    NAME integration_test_xtp_binds
+    COMMAND
+      ${Python_EXECUTABLE} -c
+      "import os; os.environ['PYTHONPATH'] = '${CMAKE_CURRENT_BINARY_DIR}/..'; from pyxtp.xtp_binds import *")
+  set_tests_properties(integration_test_xtp_binds PROPERTIES LABELS "csg;tools;votca;xtp;integration")
+  add_test(NAME integration_Compare_pyxtp_output COMMAND $<TARGET_FILE:VOTCA::votca_compare>  -f1 file1 -f2 ${REFPATH}/file2)
```

Comment:
What is file1 and file2?

Reply:
I am removing that part. I got confused about the usage